liwords-ui/src/gameroom/board_panel.tsx

```diff
@@ -236,13 +229,14 @@ export const BoardPanel = (props: Props) => {
           </Col>
         </Row>
       </div>
-      <div style={{ marginTop: 30 }}>
+      <div>
         <GameControls
-          onExchange={exchangeTiles}
-          onPass={passTurn}
-          onChallenge={challengePlay}
-          onCommit={commitPlay}
-          currentRack={props.currentRack}
+            onRecall={recallTiles}
```

Comment:
Minor thing here -- make sure your Prettier runs automatically so that all code is formatted the same (this should be two spaces)